File: swing/src/main/java/wannabe/swing/renderer/FilledThreeDSquareWithCabinetWires.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.swing.SwingProjected;

public class FilledThreeDSquareWithCabinetWires extends SwingRenderer {
  Sides sides = new Sides();
  @Override public void draw(Graphics g, SwingProjected p) {
    sides.wireSides(g, p);
    g.fill3DRect(p.left, p.top, p.size, p.size, true);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/FilledThreeDSquareWithCabinetWires.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class FilledThreeDSquareWithCabinetWires : SwingRenderer() {
  private var sides = Sides()

  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    sides.wireSides(g, p)
    g.fill3DRect(p.left, p.top, p.size, p.size, true)
  }
}
